src/tools/jetbrains-plugin.md

```diff
@@ -4,22 +4,18 @@ description: Use Dart with a variety of IDEs and editors from JetBrains.
 ---
 
 The [Dart plugin][] adds Dart support to JetBrains IDEs such as
-IntelliJ IDEA and Android Studio.
+IntelliJ IDEA, WebStorm, and Android Studio.
 IntelliJ IDEA is an intelligent Java IDE
 with support for many other languages and frameworks.
+WebStorm is an IDE based on IntelliJ IDEA
+that's focused on web development and debugging.
 Android Studio is an IDE based on IntelliJ IDEA
 that's used for Android and Flutter development.
```

Comment:
**issue:** Limit the marketing speak for JetBrains. Use active voice where possible.

**suggestion:** Link to the apps, if you want.

```suggestion
The [Dart plugin][] adds Dart support to nine of the IDEs developed by JetBrains s.r.o. 
These IDEs provide features unique to specific programming languages.
The IDEs recommended for Flutter development include:
- IntelliJ IDEA specializes in JVM-based language development.
- WebStorm specializes in JavaScript development.
- Android Studio specializes in Android and Flutter development.
```

Reply:
Thanks. I've applied the suggestions with modifications.